test/doctool/test_html.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const assert = require('assert');
+const fs = require('fs');
+
+const common = require('../common');
+const html = require('../../tools/doc/html.js');
+
+// Test data is a list of objects with two properties.
+// The file property is the file path.
+// The html property is some html which will be generated by the doctool.
+// This html will be stripped of all whitespace because we don't currently
+// have an html parser.
+const testData = [
+  {
+    'file': 'test/fixtures/sample_document.markdown',
```

Comment:
Nit: `common.fixturesDir` instead of hardcoding the path here and in three other places.